src/layout/Layout.tsx

```diff
@@ -1,11 +1,19 @@
 import { Box, Group, Text } from '@mantine/core';
 import { DesktopFrame } from './DesktopFrame';
+import { MobileFrame } from './MobileFrame';
 import layoutClasses from '../styles/Layout.module.css';
 import { IconArrowLeft } from '@tabler/icons-react';
 import { useNavigate } from 'react-router-dom';
+import { useMobile, useTablet } from '../hooks/useBreakpoint';
 
 export const Layout = ({ children }: { children: React.ReactNode }) => {
-  return <DesktopFrame>{children}</DesktopFrame>;
+  const isTablet = useTablet();
+  const isMobile = useMobile() || isTablet;
```

Comment:
Interesting that we need to use isTablet as well here. I'm fine with leaving this in. But if this needs to be done, something's up with the hook. Might want to make a ticket to fix this later.